src/PulseServiceProvider.php

```diff
@@ -52,6 +53,7 @@ protected function registerIngest(): void
         $this->app->bind(Ingest::class, fn (Application $app) => match ($app->make('config')->get('pulse.ingest.driver')) {
             'storage' => $app->make(StorageIngest::class),
             'redis' => $app->make(RedisIngest::class),
+            null, 'null' => $app->make(NullIngest::class),
```

Comment:
phpunit.xml and .env will provide us with `null`, while our first party configs often hardcode the string `'null'` to indicate the null driver should be used, as seen in `config/logging.php` and `config/broadcasting.php`.